src/CommonPool.h

```diff
@@ -28,11 +28,11 @@ class CommonPool
 
 public:
     static CommonPool *Factory (unsigned char _class, CompositePoolNode::Pointer&);
-    char const* theClassTypeLabel() const {return typeLabel.termedBuf();}
+    const SBuf &theClassTypeLabel() const { return typeLabel; }
```

Comment:
If you have the time, please rename this method to classTypeLabel() -- the old name goes against our current naming style, and all callers are already affected by the type change. Otherwise, this can be merged as is because that renaming can be viewed as out of scope.

Reply:
Sure.
Update coming up